filters/timeprefix.go

```diff
@@ -19,15 +19,15 @@ type TimePrefixFilter struct {
 
 // FilterLine prepends the current time, in square brackets with a separating
 // space, to the provided log line.
-func (f *TimePrefixFilter) FilterLine(line string) string {
+func (f TimePrefixFilter) FilterLine(line string) string {
```

Comment:
I was worried that passing the filter by value would be slow, but it's a bool & a string value, so should be fine.

Reply:
The filter was unusable without this signature; switching it to pointer receivers is a bigger change for the future maybe :(

The performance effects you mention could potentially be a thing but it's a thing that's been for some time if so

Reply:
Actually... this is that Go thing about interfaces and pointer receivers that I don't quite get yet. The test this caused a problem with is https://github.com/stripe/unilog/pull/42/files#diff-2845aa72f8c25965ab21a0741a0b0d2dR297